Add page of external resources (#18)
NASA-IMPACT/veda-docs: external-resources.qmd

---
title: "External Resources"

> subtitle: "List of resources for learning more about open source geospatial tools in Python"

Reviewer: ```suggestion
subtitle: "Learn more about tools and practice open source geospatial data science - a list of external resources to get you started"
```

---

> 

Reviewer: ```suggestion

This list is intended for (aspiring) scientists who want to get started with cloud-based, collaborative, open geospatial data analysis, or those looking to refresh their knowledge. It is by no means complete, but contains pointers to more elaborate resources. We anticipate this list to evolve as our platform and use cases evolve. Suggestions for additional resources or topics are highly welcome.

```

> ## Workflow resources

Reviewer: ```suggestion
## Master the basic workflow tools
```

* [Git - for managing code versions](https://docs.github.com/en/get-started/quickstart/github-flow)
* [Conda - for managing dependencies](https://docs.conda.io/en/latest/)
* [Jupyter - for running code](https://jupyter.org)

> ## Geospatial courses/books

Reviewer: ```suggestion
## Take a course or read a book on geospatial data analysis
```

* [Data Carpentry geospatial course](https://datacarpentry.org/geospatial-workshop/)
* [UW course on Geospatial Data Analysis with Python](https://uwgda-jupyterbook.readthedocs.io/)
* [Geographic data book - emphasis on vector data](https://geographicdata.science/book/intro.html)

> ## Geospatial docs

Reviewer: ```suggestion
## Look through nicely curated lists of tools and resources
```

* [Project Pythia - Pangeo's education hub](https://projectpythia.org/)
* [Planetary Computer](https://planetarycomputer.microsoft.com/docs/overview/about)
* [Geospatial Computing Platform library of training resources (by Python package)](https://crib.utwente.nl/manual/training-resources/README.html)

> ## Materials from hack weeks

Reviewer: ```suggestion
## Find out what a Hack Week is and see how they teach best practice
```

* [UW GeoHackWeek](https://github.com/geohackweek)
* [UW OceanHackWeek](https://github.com/oceanhackweek/)

> ## Blogs

Reviewer: ```suggestion
## Stay up-to-date through blogs
```
